.github/workflows/cd-server.yml

```diff
@@ -0,0 +1,75 @@
+name: Deploy server
+on:
+  push:
+    branches:
+      - dev
+    tags:
+      - "[v]?[0-9]+.[0-9]+.[0-9]+*"
+  workflow_dispatch:
+    inputs:
+      environment:
+        description: "Environment"
+        required: true
+        default: "enable"
```

Comment:
Sorry can I know how does this trigger setting ensure that push to dev will set DATA_ENV as nightly; while push to tag will set DATA_ENV as stable? 🤔

Reply:
Indeed, after introducing push on multiple branches, an environment manipulation step should be introduced in the workflow. I' ll push this change ASAP.

Reply:
https://github.com/ntampakas/tlsn/commit/5d9471c2645597f1b03fbf23630b97e61b69f0d9